student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Student < Person
+  attr_reader :classroom, :parent_permission
+
+  def initialize(id, age, name = 'unknown', parent_permission: true, classroom: nil)
+    super(id, name, age, parent_permission: parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] The `id` attribute does not need to be passed in the initialize method of the `Student` class because you have assigned it to a random number in the `Person` class. Since the `Student` class inherits from the `Person` class, the `id` attribute will be automatically set. When I try to create a new instance of the `Student` class, I get this error:

![image](https://github.com/Munyabelden/Oop_Library/assets/77106962/3b6b7393-c09b-44f9-8558-f598abc6fc35)

Please, don't pass the `id` attribute as an argument to the `initialize` method of the `Student` class. 👍 😎